docs/readme.txt

```diff
@@ -1,172 +1,141 @@
-                wxWidgets 3.1.0 Release Notes
-                =============================
+# wxWidgets 3.1.0 Release Notes
```

Comment:
I agree (both with making it more Markdown-ish and with keeping the name), but could we still keep the underlines for H1/H2 instead of hashes? Underlines are significantly more readable in plain text form IMO.